client/blocks/reader-combined-card/index.jsx

```diff
@@ -33,9 +33,22 @@ class ReaderCombinedCard extends React.Component {
 	}
 
 	componentDidMount() {
-		const { posts, site, feed } = this.props;
-		const feedId = get( feed, 'feed_ID' );
-		const siteId = get( site, 'ID' );
+		this.recordRenderTrack();
+	}
+
+	componentDidUpdate( nextProps ) {
+		if ( size( this.props.posts ) !== size( nextProps.posts ) ) {
+			this.recordRenderTrack();
+		}
+	}
+
+	recordRenderTrack = () => {
+		const { posts } = this.props;
+
+		// Use site ID/feed ID from the first post, because we may not have site and feed objects yet
+		const siteId = posts[ 0 ].blog_ID;
```

Comment:
might need to very the post first. it could be _state == minimal and not loaded yet.

also, are these the right properties? i don't think posts ever have a `blog_ID`, it's usually `site_ID`, assuming `is_external` is false. and the feed ID is usually `feed_ID`.